finch.yaml

```diff
@@ -143,6 +143,15 @@ provision:
 - mode: boot
   script: |
     modprobe virtiofs
+- mode: system
+  script: |
+    cat <<EOF >> /etc/sysctl.d/99-lima.conf
```

Comment:
Does this run only on `vm init` or `vm start`? Wondering if changes will be appended to the file. I also noticed that `modprobe br_netfilter` would automatically set `bridge-nf-call-iptables` to 1.

Reply:
good catch, it does append to the file on every `vm start`. it doesn't seem to affect the functionality but the file would end up getting pretty long 😅 

> I also noticed that modprobe br_netfilter would automatically set bridge-nf-call-iptables to 1.

yeah, confirmed this works for me too. good catch! will follow up with a new revision